regression/jbmc-strings/StringIndexOf/test.desc

```diff
@@ -0,0 +1,7 @@
+THOROUGH
+Test.class
+--refine-strings --function Test.check --unwind 10 --string-max-length 10 --java-assume-inputs-non-null
```

Comment:
So how about reducing the limits to, e.g., 3? Would that make the test complete more quickly?

Reply:
I try with 3 and 4 and it finishes with 1 second but with 5 it takes a long time. Then checking only 3 characters seems a bit limited. The reason I wrote this tests was to make sure the constraints I wrote for indexOf were correct, (they weren't at that point), now that we know that the constraints are correct, there won't be any bug specific to indexOf as long as we don't change them. So I think we don't have to run this test on CI, but it is still nice to have it available.

Reply:
How about you include a quick version with 3 or 4 marked as CORE, and this one in addition?